mtgjson5/set_builder.py

```diff
@@ -713,12 +713,12 @@ def build_mtgjson_card(
     mtgjson_card.color_identity = scryfall_object.get("color_identity", "")
     mtgjson_card.converted_mana_cost = scryfall_object.get("cmc", "")
     mtgjson_card.edhrec_rank = scryfall_object.get("edhrec_rank")
+    mtgjson_card.finishes = scryfall_object.get("finishes")
     mtgjson_card.frame_effects = scryfall_object.get("frame_effects", "")
     mtgjson_card.frame_version = scryfall_object.get("frame", "")
     mtgjson_card.hand = scryfall_object.get("hand_modifier")
-    mtgjson_card.has_foil = scryfall_object.get("foil")
-    mtgjson_card.has_non_foil = scryfall_object.get("nonfoil")
-
+    mtgjson_card.has_foil = "foil" in scryfall_object.get("finishes", [])
+    mtgjson_card.has_non_foil = "nonfoil" in scryfall_object.get("finishes", [])
```

Comment:
for compatibility you should check for `glossy` too

Reply:
There won't be a `hasGlossy` as we're deprecating foil/nonfoil top level (5.2.0 removal). Use finishes instead

Reply:
i'm saying that glossy cards that used to be with `hasfloil` set, won't be anybor with this simple check

this is a break that is easily avoidable

Reply:
Ahh yep, this has been changed. :D